state_machine/applications/system-check/tests/opt3001_sun_sensor_test.py

```diff
@@ -0,0 +1,71 @@
+from lib.pycubed import cubesat
+from print_utils import bold, normal
+
+def test_sun_sensor(sensor, sensor_name, sensors):
+    """
+    Tests if a sun sensor is:
+        - Detected
+        - Driver sensor coresponds to the correct physical sensor
+    """
+    if sensor is None:
+        return ("not detected", False)
+    input(f'Place the {bold}{sensor_name}{normal} board face down on the table, then press enter: ')
+    min_lux = sensor.lux()
+    for i in range(len(sensors)):
+        (other, other_name) = sensors[i]
+        if other is None:
+            continue
+        lux = other.lux()
+        if lux < min_lux and other != sensor:
+            return (f"{sensor_name} should read the lowest lux, but {other_name} read lower ({min_lux} > {lux})", False)
+    return ("success", True)
+
+def consistency_test():
```

Comment:
This is cool. It might be informative to run it on the old sun sensors/satellite as well and see how they compare.

Reply:
By making the interface the same (by doing @property for the lux call) you should be able to exactly use this function for both.